packages/cli/package.json

```diff
@@ -76,7 +76,7 @@
     "@types/formidable": "^3.4.0",
     "@types/json-diff": "^1.0.0",
     "@types/jsonwebtoken": "^9.0.1",
-    "@types/localtunnel": "^1.9.0",
+    "@types/localtunnel": "^2.0.4",
```

Comment:
Was there a need to update this dependency in the scope of this PR? If not, would not be nice if it was were done in separate PR in the future.

Reply:
The types were wrong. The call to `localtunnel` needs an `await`, but the types said otherwise. had to upgrade to get the types to match the actual implementation.